selfdrive/boardd/boardd.cc

```diff
@@ -489,7 +489,7 @@ void panda_state_thread(PubMaster *pm, std::vector<Panda *> pandas, bool spoofin
 void peripheral_control_thread(Panda *panda) {
   util::set_thread_name("boardd_peripheral_control");
 
-  SubMaster sm({"deviceState", "driverCameraState"});
+  SubMaster sm({"deviceState", "driverCameraState"}, {"deviceState", "driverCameraState"});
```

Comment:
if this is necessary, this should be in another PR merged before this one

Reply:
this really shouldn't be necessary though. let's make it the same API as the python where we default to polling on all